shared/src/libs/helpers/convert-plain-string-to-snake-case/convert-plain-string-to-snake-case.helper.ts

```diff
@@ -0,0 +1,8 @@
+const convertPlainStringToSnakeCase = (input: string): string => {
+  return input
+    .split(' ')
+    .map((word) => word.replaceAll(/-+/g, '_').toLowerCase())
+    .join('_');
+};
```

Comment:
Maybe let's use lodash.snakecase to don't give a care about any cases with wording to snake_case